src/components/Rocket.js

```diff
@@ -0,0 +1,34 @@
+import { useDispatch } from 'react-redux';
+import styles from './styles/Rocket.module.css';
+import { rocketActions } from '../redux/rocketsSlice';
+
+const Rocket = (props) => {
+  const dispatch = useDispatch();
+
+  const {
+    name, description, image, id, reserved,
+  } = props;
+  return (
+    <article>
+      <main className={styles.rocket_container}>
+        <div className={styles.image_container}>
+          <img src={image} alt="some-rocket" />
+        </div>
+        <div className={styles.rocket_details_container}>
+          <h1>{name}</h1>
+          {reserved && <span>Reserved</span>}
+          <p>{description}</p>
+          <button
+            className={reserved ? styles.reserved : styles.not_reserved}
+            type="button"
+            onClick={() => dispatch(rocketActions.createReservation(id))}
+          >
+            {reserved ? 'Cancel Reservation' : 'Reserve Rocket'}
+          </button>
+        </div>
+      </main>
+    </article>
```

Comment:
- [ ] I will implore you to design each rocket according to the project requirement. When a reservation is made, you will notice that the location and color of yours is different from the one of the requirement. Even the cancel reservation button has a transparent background.

## Project requirement
![image](https://github.com/bilalrajput09/space-travelers-app/assets/86200356/9396232b-86ad-4800-8795-301bfd4efd0d)

## Your project
![image](https://github.com/bilalrajput09/space-travelers-app/assets/86200356/43b9c1f9-b780-40c0-9ff3-48b84f45b83a)